src/vm_cpu.rs

```diff
@@ -285,6 +285,79 @@ impl VmPmuState {
             )
         }
     }
+
+    fn get_counter_csr_value(counter_index: u64) -> Option<u64> {
+        use drivers::pmu::PmuInfo;
```

Comment:
If the `use` declaration is going to get used multiple times and the risk of conflict is low, I'd just put it at the top of the file. Same for `println` below.

Reply:
Changed.